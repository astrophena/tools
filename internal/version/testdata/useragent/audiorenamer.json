{
  "GoVersion": "go1.22.3",
  "Path": "go.astrophena.name/tools/cmd/audiorenamer",
  "Main": {
    "Path": "go.astrophena.name/tools",
    "Version": "(devel)",
    "Sum": "",
    "Replace": null
  },
  "Deps": [
    {
      "Path": "github.com/dhowden/tag",
      "Version": "v0.0.0-20240417053706-3d75831295e8",
      "Sum": "h1:OtSeLS5y0Uy01jaKK4mA/WVIYtpzVm63vLVAPzJXigg=",
      "Replace": null
    }
  ],
  "Settings": [
    {
      "Key": "-buildmode",
      "Value": "exe"
    },
    {
      "Key": "-compiler",
      "Value": "gc"
    },
    {
      "Key": "-trimpath",
      "Value": "true"
    },
    {
      "Key": "CGO_ENABLED",
      "Value": "0"
    },
    {
      "Key": "GOARCH",
      "Value": "arm64"
    },
    {
      "Key": "GOOS",
      "Value": "android"
    },
    {
      "Key": "vcs",
      "Value": "git"
    },
    {
      "Key": "vcs.revision",
      "Value": "bfc4e5b495a5cfad1402fd9e30861daf6094ce62"
    },
    {
      "Key": "vcs.time",
      "Value": "2024-07-13T21:20:50Z"
    },
    {
      "Key": "vcs.modified",
      "Value": "false"
    }
  ]
}