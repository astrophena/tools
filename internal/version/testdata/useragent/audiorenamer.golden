audiorenamer/bfc4e5b4 (+https://astrophena.name/bleep-bloop)